dll/win32/advapi32/reg/reg.c

```diff
@@ -3492,7 +3492,7 @@ RegOpenUserClassesRoot(IN HANDLE hToken,
                                      NULL,
                                      0,
                                      &RequiredLength);
-    if (!NT_SUCCESS(Status) && (Status != STATUS_BUFFER_TOO_SMALL))
+    if (Status != STATUS_BUFFER_TOO_SMALL)
```

Comment:
This is bogus. You're now returning success here but no valid key handle.

Reply:
Agreed. Fixed.